pkg/controller/services.go

```diff
@@ -1663,15 +1664,17 @@ func (cont *AciController) endpointSliceUpdated(oldobj interface{}, newobj inter
 		cont.log.Error("error processing Endpointslice object: ", newobj)
 		return
 	}
-	proceed := true
-	for _, endpoint := range newendpointslice.Endpoints {
-		if endpoint.Conditions.Ready == nil || !*endpoint.Conditions.Ready {
-			proceed = false
+	if strings.ToLower(os.Getenv("CONFORMANCE_TESTS")) != "true" {
```

Comment:
Hate to nit-pick here, can we call this "wait_for_service_ep_readiness"? Also can we drive this change from config.go with the default value set to True? That way we can hook up with acc-provision.